README.md

```diff
@@ -287,11 +287,16 @@ Building Julia requires that the following software be installed:
 - **[perl]**                    — preprocessing of header files of libraries.
 - **[wget]**, **[curl]**, or **[fetch]** (FreeBSD) — to automatically download external libraries.
 - **[m4]**                      — needed to build GMP.
-- **[awk]**                     - helper tool for Makefiles.
+- **[awk]**                     — helper tool for Makefiles.
 - **[patch]**                   — for modifying source code.
 - **[cmake]** (>= 3.4.3)        — needed to build `libgit2`.
 - **[pkg-config]**              — needed to build `libgit2` correctly, especially for proxy support.
 
+On e.g. Ubuntu you can easily install them with `apt-get`:
```

Comment:
Could instead say "On Debian-based distributions (e.g. Ubuntu), [...]", but this is up to you.

Reply:
Sometimes the packages names change even among related distro families, if the command is portable, then I agree, else another line for debian could be added and some other popular distro choices?